### ISSUE TITLE
ResidenceChangedEvent -> error

### ISSUE THREAD

**Issue by bobhenl** [Zrips/Residence#697]
```
17:40:01 ERROR]: Could not pass event ResidenceChangedEvent to Residence v4.9.3.3
java.lang.NullPointerException: null
        at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener1_9.onResidenceChange(ResidencePlayerListener1_9.java:68) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1651.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener.handleNewLocation(ResidencePlayerListener.java:2166) ~[?:?]
        at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener.onPlayerMove(ResidencePlayerListener.java:2002) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1682.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1314) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.PacketPlayInFlying.a(SourceFile:122) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.PacketPlayInFlying$PacketPlayInPosition.a(SourceFile:56) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:43) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1199) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1192) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeAll(IAsyncTaskHandler.java:95) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1329) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1008) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:178) ~[patched_1.16.5.jar:git-Tuinity-"9ac33d1"]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_281]
> 
```

**Comment by Dzeivi** [comment]
#693

**Comment by Zrips** [comment]
Fixed in 4.9.3.4
